ee/clickhouse/queries/funnels/test/test_funnel_correlation.py

```diff
@@ -160,6 +171,104 @@ def test_basic_funnel_correlation_with_events(self):
         self.assertEqual(len(self._get_people_for_event(filter, "negatively_related", success=False)), 5)
         self.assertEqual(len(self._get_people_for_event(filter, "negatively_related")), 0)
 
+    @snapshot_clickhouse_queries
+    def test_funnel_correlation_with_events_and_actions(self):
```

Comment:
non-blocking

If this is explicitly checking that actions are excluded, perhaps something like `test_action_events_are_excluded_from_correlations`?